docs/adrs/integrat-vpns.md

```diff
@@ -0,0 +1,76 @@
+# Integrate vpn in k3s
```

Comment:
nit on the filename - should it be `integrate-vpns.md`? I'm not sure why the word was truncated.

Reply:
good catch, probably my mistake